com.discordapp.DiscordPTB.json

```diff
@@ -0,0 +1,170 @@
+	],
+	"finish-args": [
+		"--socket=x11",
+		"--socket=pulseaudio",
+		"--share=network",
+		"--share=ipc",
+		"--device=all",
+		"--talk-name=org.freedesktop.Notifications",
+		"--filesystem=xdg-videos:ro",
+		"--filesystem=xdg-pictures:ro",
+		"--filesystem=xdg-download",
+		"--talk-name=org.kde.StatusNotifierWatcher",
+		"--talk-name=com.canonical.AppMenu.Registrar",
+		"--talk-name=com.canonical.indicator.application",
+		"--talk-name=com.canonical.Unity.LauncherEntry"
+	],
+	"modules": [{
+			"name": "twemoji-fonts",
+			"buildsystem": "simple",
+			"sources": [{
+					"type": "file",
+					"path": "emoji-font.conf"
+				},
+				{
+					"type": "file",
+					"path": "Twemoji.ttf"
+				}
+			],
+			"build-commands": [
+				"mkdir -p /app/share/fonts/",
+				"cp Twemoji.ttf /app/share/fonts/",
+				"mkdir -p /app/etc/fonts/",
+				"cp emoji-font.conf /app/etc/fonts/local.conf",
+				"fc-cache -fv"
+			]
+		},
+		{
+			"name": "xprop",
+			"sources": [{
+				"type": "archive",
+				"url": "https://xorg.freedesktop.org/archive/individual/app/xprop-1.2.5.tar.bz2",
+				"sha256": "9b92ed0316bf2486121d8bac88bd1878f16b43bd335f18009b1f941f1eca93a1",
+				"x-checker-data": {
+					"type": "anitya",
+					"project-id": 14958,
+					"url-template": "https://xorg.freedesktop.org/archive/individual/app/xprop-$version.tar.bz2"
+				}
+			}]
+		},
+		{
+			"name": "python-setuptools_scm",
+			"buildsystem": "simple",
+			"build-commands": [
+				"pip3 install --no-deps --no-build-isolation --verbose --prefix=${FLATPAK_DEST} ."
+			],
+			"sources": [{
+				"type": "archive",
+				"url": "https://files.pythonhosted.org/packages/57/38/930b1241372a9f266a7df2b184fb9d4f497c2cef2e016b014f82f541fe7c/setuptools_scm-6.0.1.tar.gz",
+				"sha256": "d1925a69cb07e9b29416a275b9fadb009a23c148ace905b2fb220649a6c18e92"
+			}]
+		},
+		{
+			"name": "python-xlib",
+			"buildsystem": "simple",
+			"build-commands": [
+				"pip3 install --no-deps --no-build-isolation --verbose --prefix=${FLATPAK_DEST} ."
+			],
+			"sources": [{
+				"type": "archive",
+				"url": "https://files.pythonhosted.org/packages/40/9c/107e22f637d33434404c07d69112b9d26b76ee0dd4dd705131ab6cdcc818/python-xlib-0.31.tar.gz",
+				"sha256": "74d83a081f532bc07f6d7afcd6416ec38403d68f68b9b9dc9e1f28fbf2d799e9",
+				"x-checker-data": {
+					"type": "pypi",
+					"name": "python-xlib"
+				}
+			}]
+		},
+		{
+			"name": "discord-ptb",
+			"buildsystem": "simple",
+			"build-commands": [
+				"install -Dm755 discord-ptb.sh /app/bin/discord-ptb",
+				"install -Dm755 set-gtk-dark-theme.py /app/bin",
+				"install -Dm755 disable-breaking-updates.py /app/bin",
+				"mv DiscordPTB /app/discord-ptb",
+				"chmod +x /app/discord-ptb/DiscordPTB",
+				"install -d /app/share/applications",
+				"sed -e 's/Icon=discord-ptb/Icon=com.discordapp.DiscordPTB/' -e 's|Exec=/usr/share/discord-ptb/DiscordPTB|Exec=discord-ptb|' /app/discord-ptb/discord-ptb.desktop > /app/share/applications/${FLATPAK_ID}.desktop",
```

Comment:
you should use `desktop-file-edit` instead.

Reply:
done